services/app-web/src/pages/StateSubmission/StateSubmissionForm.test.tsx

```diff
@@ -445,6 +446,69 @@ describe('StateSubmissionForm', () => {
             const loading = await screen.findByText('System error')
             expect(loading).toBeInTheDocument()
         })
+
+        it('shows a generic error when updating submission fails', async () => {
+            const mockSubmission = mockDraftHealthPlanPackage({
+                submissionDescription:
+                    'A real submission but updated something',
+            })
+            const formData = base64ToDomain(
+                mockSubmission.revisions[0].node.formDataProto
+            )
+            if (formData instanceof Error) throw Error
```

Comment:
I could see this unwrap proto code being its own test helper

> const formData = base64ToDomain(mockSubmission.revisions[0].node.formDataProto)
if (formData instanceof Error) throw Error